userapi/internal/api.go

```diff
@@ -131,6 +131,11 @@ func (a *UserInternalAPI) setFullyRead(ctx context.Context, req *api.InputAccoun
 }
 
 func (a *UserInternalAPI) PerformAccountCreation(ctx context.Context, req *api.PerformAccountCreationRequest, res *api.PerformAccountCreationResponse) error {
+	serverName := req.ServerName
```

Comment:
We should probably validate this with the config as a sanity check at internal API boundaries like this.

Reply:
Yeah, these little bits in the user API are just stubs to keep things working until we've got columns in the tables to store server names in.